### ISSUE TITLE
.d81 image not working on C128D in 128 mode

### ISSUE THREAD

**Issue by jduncalf** [pi1541/Pi1541#242]
Hi, .d81 image not working on C128D in 128 mode. Doesn't retrieve a directory listing and bounces between DAT 0 & 1. Tried recommended D1581 ROM and VICE D1581 ROM. 

Same .d81 image works great in 64 mode.

Thanks.

**Comment by DeeKay64** [comment]
Yeah, Pi1541 only really works in 64 mode as 1581 (and that really well!), c128 mode is a no go since it does not (or only partially) emulate c128 specific features that the 1571 and 1581 offer such as 2MHz mode, Fast Serial or Burst Mode.
Had to find that out the hard way when we made Colour Spectrum for the c128. Got an U2+ instead because 1581 (and 1571!) emulation was just about to be released for that.. And WORKED!
